gammapy/data/tests/test_data_store.py

```diff
@@ -132,6 +132,100 @@ def test_datastore_copy_obs_subset(tmp_path, data_store):
     assert len(substore.hdu_table) == 2
 
 
+@requires_data()
+@pytest.mark.xfail
+def test_minimal_datastore(tmp_path, data_store):
+    """"Check that a standard analysis runs on a minimal datastore,
+        ie, with only the mandatory keywords defined on the obs and hdu index tables"""
+
+    from gammapy.datasets import MapDataset
+    from gammapy.maps import MapAxis, WcsGeom
+    from gammapy.makers import MapDatasetMaker, SafeMaskMaker, FoVBackgroundMaker
+    from regions import CircleSkyRegion
+    from astropy.coordinates import SkyCoord
+    import astropy.units as u
+
+    # Create a minimal datastore from 2 Crab runs:
+    obs_id = data_store.obs_table[data_store.obs_table["OBJECT"] == "Crab Nebula"][
+        "OBS_ID"
+    ]
+    data_store.copy_obs(obs_id=obs_id[:2], outdir=tmp_path)
+    datastore1 = DataStore.from_dir(tmp_path)
+    table1 = datastore1.obs_table
+    col_mandatory = ["OBS_ID", "RA_PNT", "DEC_PNT", "TSTART", "TSTOP"]
+
+    # col_mandatory.append('DEADC') TODO: uncommenting this should solve the xfail.
+
+    remove_cols = list(set(table1.colnames) - set(col_mandatory))
+    table1.remove_columns(remove_cols)
+    meta_mandatory = {
+        "HDUDOC": "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats",
+        "HDUVERS": 0.2,
+        "HDUCLASS": "GADF",
+        "HDUCLAS1": "INDEX",
+        "HDUCLAS2": "OBS",
+    }
+    table1.meta = meta_mandatory
+    table2 = datastore1.hdu_table
+    table2.remove_column("SIZE")
+    meta_hdu_mandatory = {
+        "HDUDOC": "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats",
+        "HDUVERS": 0.2,
+        "HDUCLASS": "GADF",
+        "HDUCLAS1": "INDEX",
+        "HDUCLAS2": "HDU",
+    }
+    table2.meta = meta_hdu_mandatory
+    table1.write(tmp_path / "obs-index.fits.gz", overwrite=True)
+    table2.write(tmp_path / "hdu-index.fits.gz", overwrite=True)
+
+    datastore_minimal = DataStore.from_dir(tmp_path)
+
+    # Run a standard Analysis
+    energy_axis = MapAxis.from_energy_bounds(
+        1, 10, nbin=3, per_decade=False, unit="TeV", name="energy"
+    )
+    geom = WcsGeom.create(
+        skydir=(83.633, 22.014),
+        binsz=0.5,
+        width=(2, 2),
+        frame="icrs",
+        proj="CAR",
+        axes=[energy_axis],
+    )
+    observations = datastore_minimal.get_observations(obs_id[:2])
+    maker = MapDatasetMaker()
+    offset_max = 2.3 * u.deg
+    maker_safe_mask = SafeMaskMaker(methods=["offset-max"], offset_max=offset_max)
+    circle = CircleSkyRegion(
+        center=SkyCoord("83.63 deg", "22.14 deg"), radius=0.2 * u.deg
+    )
+    exclusion_mask = ~geom.region_mask(regions=[circle])
+    maker_fov = FoVBackgroundMaker(method="fit", exclusion_mask=exclusion_mask)
+
+    stacked = MapDataset.create(geom=geom, name="crab-stacked")
+    for obs in observations:
+        # First a cutout of the target map is produced
+        cutout = stacked.cutout(
```

Comment:
Minor: maybe simplify and remove the cutout, I think performance wise it shouldn't really matter here...